alternator/rjson.cc

```diff
@@ -158,20 +166,18 @@ rjson::value& get(rjson::value& value, std::string_view name) {
     // Luckily, the variant taking a GenericValue doesn't share this bug,
     // and we can create a string GenericValue without copying the string.
     auto member_it = value.FindMember(rjson::value(name.data(), name.size()));
-    if (member_it != value.MemberEnd())
+    if (member_it != value.MemberEnd()) {
         return member_it->value;
-    else {
-        throw rjson::error(format("JSON parameter {} not found", name));
     }
+    throw missing_parameter(name);
```

Comment:
Because 

'''
    return ...;
  } else {
    throw  ...;
  } 
  ... 
}
'''
implies that there is room for anything after the conditional block. 
A straight throw at end of function is imho clearer.

Reply:
I don't think it implies anything - you can very well have if()..else and in *both* of them call a function that stops the execution (return or throw). It's just a matter of style. But ok, I don't mind if you change this.